### ISSUE TITLE
url = os.path.realpath(url) Bug

### ISSUE THREAD

**Issue by macmichael01** [dziegler/django-css#24]
Compressor seems to break on line 67 of compressor.**init**.py
which is this:
url = os.path.realpath(url)

If I comment this line out, it works.

I am using python 2.6 with virtualenv + pip. I have some other projects that I am working on that aren't using virtualenv and I don't experience the above issue. Only with virtualenv.

**Comment by dhirsch** [comment]
That fixed my problem as well, same environment

**Comment by humphrey** [comment]
Yes, I also had to comment out that line...  Adding print statements before and after that line gave me:

http://my.domain/static/sms/css/screen.css
C:\my\project\dir\http:\pianos1st.localhost\static\sms\css\screen.css

I think that doesn't work under windows :-)

**Comment by drozzy** [comment]
Wow, thanks - that fixed my bug:
https://github.com/dziegler/django-css/issues#issue/32

**Comment by drozzy** [comment]
Any chance of actually fixing this in source - so I don't have to hack it everytime I install it?
